recipes/recipes_emscripten/gtest/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -ex
+
+# Build and install dynamic library
+if [ ! -d "build" ]; then
```

Comment:
You don't need to change this, but just for future reference, you can also do this in a single line
```
mkdir -p build
```
which will show no error if the directory already exists.